gunrock/app/hits/hits_test.cuh

```diff
@@ -21,19 +21,23 @@ namespace hits {
 /******************************************************************************
  * HITS Testing Routines
  *****************************************************************************/
-
+// TODO document this
```

Comment:
What needs additional docs?

Reply:
This was referring to the function parameter documentation, which was already done and I just forgot to remove the comment. Fixed.